modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/diagnostic/pagelocktracker/PageLockDump.java

```diff
@@ -16,17 +16,50 @@
 
 package org.apache.ignite.internal.processors.cache.persistence.diagnostic.pagelocktracker;
 
+import org.apache.ignite.internal.processors.cache.persistence.diagnostic.pagelocktracker.dumpprocessors.ToStringDumpHelper;
+
 /**
- * Interface for page lock tracker structures dump.
+ * Base page lock tracker structures dump.
  */
-public interface PageLockDump {
-    /**
-     * @param dumpProcessort Apply dump processor.
-     */
-    void apply(DumpProcessor dumpProcessort);
-
-    /**
-     * @return Dump creation time.
-     */
-    long time();
+public abstract class PageLockDump {
```

Comment:
Can we make it package-private?
Can we make most of these classes package-private?